--- anjuta-3.34.0/configure.ac	2022-02-21 11:15:18.279900520 -0700
+++ anjuta-3.34.0/configure.ac.new	2022-02-21 11:15:58.373897320 -0700
@@ -813,10 +813,7 @@
 AC_SUBST(SYMBOL_DB_SHM)
 
 dnl On Linux, need librt for shm_open/shm_unlink
-bck_LIBS="$LIBS"
-AC_SEARCH_LIBS(shm_open, rt, [SHM_LIBS="$LIBS"], [AC_MSG_ERROR([Failed to find library with shm_open()])])
-AC_SUBST(SHM_LIBS)
-LIBS="$bck_LIBS"
+LIBS="$LIBS -lrt"
 
 dnl Test using autotest
 dnl -----------------------------
