From 6a337de43686989fe3db07aed3fd5317b9754a8a Mon Sep 17 00:00:00 2001
From: Gilles Dartiguelongue <eva@gentoo.org>
Date: Thu, 4 Oct 2018 10:31:42 +0200
Subject: [PATCH 2/3] Install missing tool for sandboxing

---
 client/Makefile.am | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/client/Makefile.am b/client/Makefile.am
index 0fb91d3..06b48df 100644
--- a/client/Makefile.am
+++ b/client/Makefile.am
@@ -24,7 +24,7 @@ endif
 
 bin_PROGRAMS = boinc_client boinccmd
 if !OS_WIN32
-pkglibexec_PROGRAMS = switcher
+pkglibexec_PROGRAMS = switcher setprojectgrp
 endif
 
 boinccmd_SOURCES = boinc_cmd.cpp
@@ -127,6 +127,10 @@ switcher_SOURCES = switcher.cpp
 switcher_LDFLAGS = $(AM_LDFLAGS) -L../lib
 switcher_LDADD = $(LIBBOINC)
 
+setprojectgrp_SOURCES = setprojectgrp.cpp
+setprojectgrp_LDFLAGS = $(AM_LDFLAGS) -L../lib
+setprojectgrp_LDADD = $(LIBBOINC)
+
 ## since we are using libtool we need some magic to get boinc and boinc_client
 ## to both be installed properly.  The next two rules do that...
 all-local: boinc$(EXEEXT)
-- 
2.18.0

