From 5b979fa7f75557b7e246a58290f70601e31db9b9 Mon Sep 17 00:00:00 2001
From: Gilles Dartiguelongue <eva@gentoo.org>
Date: Thu, 4 Oct 2018 11:13:35 +0200
Subject: [PATCH 3/3] Add missing config.h include in setprojectgrp.cpp

---
 client/setprojectgrp.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/client/setprojectgrp.cpp b/client/setprojectgrp.cpp
index 608410d..fcf64d2 100644
--- a/client/setprojectgrp.cpp
+++ b/client/setprojectgrp.cpp
@@ -23,6 +23,8 @@
 //
 // setprojectgrp runs setuid boinc_master and setgid boinc_project
 
+#include "config.h"
+
 #include <unistd.h>
 #include <grp.h>
 #include <cstdio>
-- 
2.18.0

