--- fritzing-app-0.9.3b.orig/phoenix.pro	2017-12-05 21:51:42.961868542 -0600
+++ fritzing-app-0.9.3b/phoenix.pro	2017-12-05 21:52:28.530864269 -0600
@@ -160,60 +160,7 @@
 RC_FILE = fritzing.rc
 RESOURCES += phoenixresources.qrc
 
-
-# Fritzing is using libgit2 since version 0.9.3
-
-LIBGIT2INCLUDE = ../libgit2/include
-exists($$LIBGIT2INCLUDE/git2.h) {
-    message("found libgit2 include path at $$LIBGIT2INCLUDE")
-}
-else {
-    message("Fritzing requires libgit2")
-    message("Build it from the repo at https://github.com/libgit2")
-    message("See https://github.com/fritzing/fritzing-app/wiki for details.")
-
-    error("libgit2 include path not found in $$LIBGIT2INCLUDE")
-}
-
-INCLUDEPATH += $$LIBGIT2INCLUDE
-
-win32 {
-    contains(QMAKE_TARGET.arch, x86_64) {
-            LIBGIT2LIB = ../libgit2/build64
-    }
-    else {
-            LIBGIT2LIB = ../libgit2/build32
-    }
-
-    exists($$LIBGIT2LIB/git2.lib) {
-        message("found libgit2 library in $$LIBGIT2LIB")
-    }
-    else {
-        error("libgit2 library not found in $$LIBGIT2LIB")
-    }
-}
-
-unix {
-    LIBGIT2LIB = ../libgit2/build
-    macx {
-        exists($$LIBGIT2LIB/libgit2.dylib) {
-            message("found libgit2 library in $$LIBGIT2LIB")
-        }
-        else {
-            error("libgit2 library not found in $$LIBGIT2LIB")
-        }
-    }
-    !macx {
-        exists($$LIBGIT2LIB/libgit2.so) {
-            message("found libgit2 library in $$LIBGIT2LIB")
-        }
-        else {
-            error("libgit2 library not found in $$LIBGIT2LIB")
-        }
-    }
-}
-
-LIBS += -L$$LIBGIT2LIB -lgit2
+LIBS += -lgit2
 
 include(pri/kitchensink.pri)
 include(pri/mainwindow.pri)
