diff -Nur fritzing-app-0.9.3b/src/version/partschecker.cpp fritzing-app-0.9.3b-new/src/version/partschecker.cpp
--- fritzing-app-0.9.3b/src/version/partschecker.cpp	2016-06-01 13:48:27.000000000 -0600
+++ fritzing-app-0.9.3b-new/src/version/partschecker.cpp	2018-02-16 18:45:38.105664664 -0700
@@ -121,7 +121,7 @@
     /**
      * Connect to the remote.
      */
-    error = git_remote_connect(remote, GIT_DIRECTION_FETCH, &callbacks);
+    error = git_remote_connect(remote, GIT_DIRECTION_FETCH, &callbacks, nullptr, nullptr);
     if (error) {
         partsCheckerResult.partsCheckerError = PARTS_CHECKER_ERROR_REMOTE;
         partsCheckerResult.errorMessage = QObject::tr("Unable to access network site for '%1'. %2").arg(repoPath).arg(sBoilerPlate1);
