diff --unified --new-file fritzing-app-0.9.3b.orig/fritzing.desktop fritzing-app-0.9.3b/fritzing.desktop
--- fritzing-app-0.9.3b.orig/fritzing.desktop	2017-12-08 22:25:07.115134159 -0600
+++ fritzing-app-0.9.3b/fritzing.desktop	2017-12-08 22:28:04.139148164 -0600
@@ -3,7 +3,7 @@
 Name=Fritzing
 GenericName=Fritzing
 Comment=Electronic Design Automation software
-Exec=Fritzing
+Exec=/usr/bin/fritzing.sh
 Icon=icons/fritzing_icon.png
 Terminal=false
 Type=Application
diff --unified --new-file fritzing-app-0.9.3b.orig/fritzing.sh fritzing-app-0.9.3b/fritzing.sh
--- fritzing-app-0.9.3b.orig/fritzing.sh	1969-12-31 18:00:00.000000000 -0600
+++ fritzing-app-0.9.3b/fritzing.sh	2017-12-08 22:33:03.177171821 -0600
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Not sure if it's a compile-time screwup on my part, a bug, or intentional,
+# but Fritzing looks for it's parts in whatever directory you're in when you
+# start it, ignoring the -f path option. Work-around-time!
+
+cd /usr/share/fritzing/parts/
+/usr/bin/Fritzing
Common subdirectories: fritzing-app-0.9.3b.orig/help and fritzing-app-0.9.3b/help
Common subdirectories: fritzing-app-0.9.3b.orig/pri and fritzing-app-0.9.3b/pri
Common subdirectories: fritzing-app-0.9.3b.orig/resources and fritzing-app-0.9.3b/resources
Common subdirectories: fritzing-app-0.9.3b.orig/sketches and fritzing-app-0.9.3b/sketches
Common subdirectories: fritzing-app-0.9.3b.orig/src and fritzing-app-0.9.3b/src
Common subdirectories: fritzing-app-0.9.3b.orig/tools and fritzing-app-0.9.3b/tools
Common subdirectories: fritzing-app-0.9.3b.orig/translations and fritzing-app-0.9.3b/translations
